app/models/offence.rb

```diff
@@ -14,12 +14,18 @@ class Offence < ActiveRecord::Base
   auto_strip_attributes :description, squish: true, nullify: true
 
   belongs_to :offence_class
+  belongs_to :fee_band
   has_many :claims, -> { active }, class_name: Claim::BaseClaim, foreign_key: :offence_id, dependent: :nullify
+  has_many :offence_fee_schemes
+  has_many :fee_schemes, through: :offence_fee_schemes
```

Comment:
❓ Can a given offence be associated with multiple fee schemes?

Reply:
@lostie should discuss as i had a similar question but looks like a given offence can be applicable to a scheme and version and at different `active` dates I.e. AGFS 9 and LGFS 9. and I am guess we may eventually have LGFS scheme 10